FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -21,7 +21,8 @@ @implementation FBSnapshotTestCase
 - (void)setUp
 {
   [super setUp];
-  _snapshotController = [[FBSnapshotTestController alloc] initWithTestName:NSStringFromClass([self class])];
+  NSString *testName = [[NSStringFromClass([self class]) componentsSeparatedByString:@"."] lastObject];
```

Comment:
What about going even further and allow the user to optionally decide the folder's name?